packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceCreateManifest.test.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import { fail } from 'assert';
+import { expect } from 'chai';
+import * as fs from 'fs';
+import util = require('util');
+import * as sinon from 'sinon';
+import { createSandbox } from 'sinon';
+import * as vscode from 'vscode';
+import {
+  forceCreateManifest
+} from '../../../src/commands';
+import { nls } from '../../../src/messages';
+
+const classPath = '/force-app/main/default/classes/';
+const class1 = 'Apex1';
+const class2 = 'Apex2';
+const uri1 = vscode.Uri.parse(classPath + class1);
+const uri2 = vscode.Uri.parse(classPath + class2);
+const manifest = `<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+  <types>
+    %s
+    <name>ApexClass</name>
+  </types>
+</Package>`;
+
+describe('Force Create Manifest', () => {
+  describe('Happy Paths', () => {
+
+    let env: sinon.SinonSandbox;
+    let openDocSpy: sinon.SinonSpy;
+    let inputBoxSpy: sinon.SinonStub;
+
+    before(() => {
+      env = createSandbox();
+    });
+
+    beforeEach(() => {
+      inputBoxSpy = env.stub(vscode.window, 'showInputBox');
+      openDocSpy = env.spy(vscode.workspace, 'openTextDocument');
```

Comment:
Just a reminder, to rename to showInputBoxStub, and openTextDocumentSpy

Reply:
Yes, working on that currently!